common/djangoapps/microsite_configuration/backends/database.py

```diff
@@ -126,6 +127,8 @@ class DatabaseMicrositeTemplateBackend(BaseMicrositeTemplateBackend):
     """
     Specialized class to pull templates from the database
     """
+    def get_template_path(self, relative_path, **kwargs):
+        return relative_path
```

Comment:
that is what I was trying in my dev environment, using both backends with the database defined microsites. That is how I found the weird behavior. Now other thing we could do, is add a third backend for templates, that handles both ends, e.g. get the template from the db and if not, resort to the template in the microsite template_dir, and only if both are unexistent, then return None and leave mako work its way thought the paths in the loader